internal/log/log.go

```diff
@@ -24,9 +25,14 @@ func init() {
 }
 
 // ReplaceGlobalLogger replace the logger and inject it globally
-func ReplaceGlobalLogger(enableJsonOutput bool, verbosityLevel string) error {
+func ReplaceGlobalLogger(enableJsonOutput bool, verbosityLevel string, enableColorEncoder bool) error {
 	config := zap.NewProductionConfig()
 
+	// Colour coding
+	if enableColorEncoder {
+		config.EncoderConfig.EncodeLevel = zapcore.CapitalColorLevelEncoder
+	}
```

Comment:
I think we can add here an else statement setting the config.EncoderConfig.EncodeLevel to zapcore.CapitalLevelEncoder because right now we are using a lower case by default

Reply:
Fixed here https://github.com/NethermindEth/juno/pull/335/commits/148bfa04421758373996fe001dbe16941512c4b4. Thank you!